content/api/users/code_snippets/api-user-create.rb

```diff
@@ -6,4 +6,4 @@
 
 dog = Dogapi::Client.new(api_key, app_key)
 
-dog.create_user(: handle => 'test@datadoghq.com', : name => 'test user')
\ No newline at end of file
+dog.create_user(: id => 'test@datadoghq.com', : name => 'test user')
```

Comment:
Could you change this to:

```ruby
dog.create_user(:id => 'test@datadoghq.com', :name => 'test user')
```

or even better:

```ruby
dog.create_user(id: 'test@datadoghq.com', name: 'test user')
```

Ruby has two hash syntaxes which is kinda weird, but the second one is preferred for symbol keys as of [Ruby 1.9](http://www.rubydoc.info/gems/rubocop/0.45.0/RuboCop/Cop/Style/HashSyntax).

Reply:
oooo, nice comment. i did not know this and just got confused when i saw the two